apps/web-giddh/src/app/home/components/cr-dr-list/cr-dr-list.component.ts

```diff
@@ -96,6 +101,18 @@ export class CrDrComponent implements OnInit, OnDestroy {
                 this.activeCompany = activeCompany;
             }
         });
+
+        this.store.pipe(select(appStore => appStore.settings.branches), takeUntil(this.destroyed$)).subscribe(response => {
+            if (response) {
+                this.isCompany = this.generalService.currentOrganizationType !== OrganizationType.Branch && response?.length > 1;
```

Comment:
```suggestion
                this.isCompany = this.generalService.currentOrganizationType !== OrganizationType.Branch && response.length > 1;
```

check everywhere same code